### PR TITLE
Add reroll for coop

### PR DESCRIPTION
**Pull request by bgniebs**
Not gonna add it to vs. till we have team limiting

### CODE REVIEW

File: games/telewave/telewave.py [hunk 1]
```diff
@@ -83,21 +83,26 @@ async def play_coop(bot, ctx, team):
...
     while turns:
-        await wait(ctx, team, extra_turn)
-        prompt = get_prompt()
-        answer = random.randint(0, 100)
-        await display(
-            ctx, team, None, prompt, max_score, turns,
-            text=f"\u200B\n**{team.psychic.mention}** is thinking of a clue...\n",
-            thumbnail=str(team.psychic.avatar_url)
-        )
+        while true:
```
Comment (vsporeddy) [inline games/telewave/telewave.py:86]: True

File: games/telewave/telewave.py [hunk 2]
```diff
@@ -83,21 +83,26 @@ async def play_coop(bot, ctx, team):
...
+            )
 
-        clue = await get_clue(bot, ctx, team.psychic, prompt, answer)
-        await display(
-            ctx, team, None, prompt, max_score, turns,
-            text=f"Clue: ```{clue}```",
-            thumbnail=str(team.psychic.avatar_url)
-        )
+            clue = await get_clue(bot, ctx, team.psychic, prompt, answer)
+            if (clue == "__###_reroll_me_##__"):
```
Comment (vsporeddy) [inline games/telewave/telewave.py:97]: don't need parentheses

File: games/telewave/telewave.py (hunk 1 above)
Comment (AaronAntone) [reply games/telewave/telewave.py]: not False:

File: Dockerfile [hunk 3]
```diff
@@ -4,4 +4,5 @@ RUN pip3 install discord
 RUN pip3 install nltk
 RUN pip3 install python-dotenv
 RUN pip3 install pynacl
+RUN pip3 install gino
```
Comment (vsporeddy) [inline Dockerfile:7]: Good catch, thanks for this!

File: games/telewave/telewave.py [hunk 4]
```diff
@@ -45,7 +46,7 @@ async def start(bot, ctx, players, game_mode):
 
     if game_mode == "coop":
         team = TelewaveTeam(
-            players, name=random.choice(team_names), score=0, cgr=await cgr.get_average_rating(players, "telewave")
+            players, name=random.choice(team_names), score=0, cgr=await cgr.get_average_rating(players, "telewave"), rerolls=2
```
Comment (vsporeddy) [inline games/telewave/telewave.py:49]: We should move this to config

**Comment by vsporeddy** [comment]
lgtm

**Review by TimmahC — APPROVED** [review]
lgtm

**Pull request merged**